src/commands/config/set.js

```diff
@@ -64,9 +62,9 @@ const setNestedConfigProperty = (config, path, value) => {
 };
 
 const attemptWriteToFile = (newConfig, value, dotNotation) => {
-	const writeSuccess = setConfig(newConfig);
+	const writeSuccess = setConfig(process.env.XDG_CONFIG_HOME, newConfig);
```

Comment:
If `setConfig` will always be given `process.env.XDG_CONFIG_HOME`, why not just access it inside the function?

Reply:
I think it's a bit weird to access the env from the util, and it makes the test harder.